src/index.js

```diff
@@ -1,126 +1,114 @@
-import React, { Component } from 'react'
-import PropTypes from 'prop-types'
+import React, { useEffect, useReducer, useRef } from "react";
```

Comment:
Can you run `yarn format` and commit? That will avoid (among others) the `'` to `"` and added semicolons (and subsequent noisy diff the next time we commit).